drift/lib/src/runtime/query_builder/components/order_by.dart

```diff
@@ -23,9 +37,21 @@ class OrderingTerm extends Component {
   /// The ordering mode (ascending or descending).
   final OrderingMode mode;
 
+  /// How to order NULLs.
+  /// When [nullsOrder] is [null], then it's ignored.
+  ///
+  /// Note that this feature are only available in sqlite3 version `3.30.0` and
+  /// newer. When using `sqlite3_flutter_libs` or a web database, this is not
+  /// a problem.
+  final NullsOrder? nullsOrder;
+
   /// Creates an ordering term by the [expression] and the [mode] (defaults to
   /// ascending).
-  OrderingTerm({required this.expression, this.mode = OrderingMode.asc});
+  OrderingTerm({
+    required this.expression,
+    this.mode = OrderingMode.asc,
+    this.nullsOrder,
+  });
 
   /// Creates an ordering term that sorts for ascending values of [expression].
   factory OrderingTerm.asc(Expression expression) {
```

Comment:
I think it would be great if the same named parameter could be added to the `asc` and `desc` factories as well.